# upstream: yes

From dd922c09bbe5deb808152140679673e8dca3210d Mon Sep 17 00:00:00 2001
From: Eli Schwartz <eschwartz@archlinux.org>
Date: Tue, 21 Jan 2020 23:55:44 -0500
Subject: [PATCH] fix incorrect install dir for fish completion

fish completions should never be installed to share/fish/completions/ as
that directory is reserved exclusively for completions shipped as part
of the fish source code.

Use the same vendor_completions.d/ directory which the default fish
configuration uses.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 17f12efd..5728a23f 100755
--- setup.py
+++ setup.py
@@ -28,7 +28,7 @@
 commands = ['ponysay', 'ponythink', 'ponysay-tool']
 
 shells = [('bash', '/usr/share/bash-completion/completions/ponysay', 'GNU Bash'),
-          ('fish', '/usr/share/fish/completions/ponysay.fish', 'Friendly interactive shell'),
+          ('fish', '/usr/share/fish/vendor_completions.d/ponysay.fish', 'Friendly interactive shell'),
           ('zsh', '/usr/share/zsh/site-functions/_ponysay', 'zsh')]
 
 mansections = [('ponysay', '6'),
